modules/tensor_mechanics/src/materials/ADLAROMANCEStressUpdateBase.C

```diff
@@ -545,6 +545,85 @@ ADLAROMANCEStressUpdateBase::computeResidual(const ADReal & effective_trial_stre
   return total_rom_effective_strain_inc - scalar;
 }
 
+Real
+ADLAROMANCEStressUpdateBase::computeCreepStrainRate(const ADReal & effective_trial_stress)
+{
+  // Update new stress
+  auto trial_stress_mpa = effective_trial_stress * 1.0e-6;
+  ADReal dtrial_stress_dscalar = 0.0;
+
+  // Update stress if strain is being applied, i.e. non-testing simulation
+  _input_values[_stress_input_index] = trial_stress_mpa;
```

Comment:
This is changing member data, maybe we can change some of these functions not to use member data so that you can make this function and those functions const.  Could you make a test for this using tensor_mechanics/test/tests/rom_stress_update/2drz.i.  You might need to run it for more time.  I have an example test for the p91 rom in blackbear_laromance that would be a good example for a creep loading history.  It might need to be changed to work with the stress range of the SS316 model